AcornSat.Core/Model/DataSet.cs

```diff
@@ -126,17 +126,18 @@ public string WarmingIndexDescription
 
             if (WarmingIndex == null)
             {
-                return $"Warming index: the temperature difference between the average of the last 20 years of maximum temperatures compared and the average of the first half of the dataset.";
+                return $"The warming index is the temperature difference between the average of the last 20 years of maximum temperatures compared and the average of the first half of the dataset.";
```

Comment:
could this say "compared _with_ the average" instead of "compared _and_ the average"?